.github/workflows/pythonpackage.yml

```diff
@@ -93,10 +93,12 @@ jobs:
       run: |
         echo "RSPM=\"https://packagemanager.rstudio.com/cran/__linux__/$(lsb_release -sc)/latest\"" >> $GITHUB_ENV
     - name: Install R dependencies
+      if: matrix.os != 'windows-latest'
```

Comment:
Why skip the optional R packages for which rpy2 includes wrappers?

Reply:
(I've been writing a comment regarding this but you beat me to it.) With the `r_home` fix, the GitHub Action builds Windows package now, but hangs during "Test with R deps" (I waited 30 minutes on it). So, it's a temporary measure till I figure out what's going on.

Reply:
A follow-up. I tested `pytest rpy2\tests\robjects\lib` locally and that works fine (with version mismatch warnings). So maybe it has something to do with these `--cov` options?

Reply:
`--cov` will track test coverage. Any chance you could test locally (you'll need the Python packages `codecov` and `pytest-cov`)?

Reply:
yep, it hangs:

```
(.venv) [...] >pytest --cache-clear --cov-append --cov-report=xml --cov-report=term --cov=rpy2.rinterface_lib --cov=rpy2.rinterface --cov=rpy2.rlike --cov=rpy2.robjects --cov=rpy2.robjects.lib rpy2/tests/robjects/lib
=================================================================== test session starts ====================================================================
platform win32 -- Python 3.9.7, pytest-7.1.2, pluggy-1.0.0
rootdir: C:\Users\tikum\Documents\Python\rpy2
plugins: cov-3.0.0
collecting ... 
```

It has been sitting there for the past 30 min.

Reply:
Thanks.

`@pytest.mark.skipif` or `@pytest.mark.xfail` specifically on Windows would be fine. Or remove `--cov=rpy2.robjects.lib` from the command line calling `pytest`.

One argument to just drop the package installation would be that `rpy2` on native Windows is on life support anyways, but the more test are removed the faster the code will drift out of working on Windows.

Reply:
Let me look into the pytest control and will make another commit.

"rpy2 on native Windows is on life support anyways"

I'd call it "community support"  :-D 

As far as I can tell, it's working great, doing everything I've asked it for. 

BTW, as long as I'm at my current job, I wouldn't mind helping you to keep `rpy2` compatible with Windows R.